src/cli.ts

```diff
@@ -43,6 +23,9 @@ const cli = meow(`
       alias: 'f',
       type: 'string',
       isRequired: false
+    },
+    help: {
+      alias: 'h'
```

Comment:
shouldn't `version` be here too? also I noticed `-v` yields help and not version

Reply:
The `--version` works automatically by meow. But now when I tested it, `-v` does not work should I close `autoVersion` and write custom logic for it.

Reply:
open a follow up issue for it please and for now remove from help messages references for `-v`